client/src/login/Login.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import axios from 'axios';
+
+class Login extends React.Component {
+  
+  state = {
+    username: '',
+    password: ''
+  }
+
+  onInputChange = event =>  {
+    const { name, value } = event.target;
+    this.setState({ 
+      [name]: value
+    })
+  }
+
+  onSubmit = event => {
+    event.preventDefault();
+    const endpoint = "http://localhost:9090/api/login"
+    axios
+      .post(endpoint, this.state)
+      .then(res => {
+        console.log(res.data)
+        localStorage.setItem('jwt', res.data.token)
+        this.props.history.push('/users')
+      })
+      .catch(err => {
+        console.error(err)
+      })
+  }
+
+  render(){
+    return (
+      <>
+        <h2>Login</h2>
+        <form onSubmit={this.onSubmit}>
+          <label htmlFor="username">Username</label>
+          <input 
+            name="username"
+            type="text"
+            id="username"
+            value={this.state.username}
+            onChange={this.onInputChange}
+          />
+          <label htmlFor="password">Password</label>
+          <input 
+            name="password"
+            type="text"
+            id="password"
+            value={this.state.password}
+            onChange={this.onInputChange}
+          />
+          <button type="submit">Login</button>
+        </form>
+      </>
```

Comment:
The Login component looks nearly identical to the Signup one; do you think you could combine them into one component (without giving up any of the current differences)?